packages/istanbul-lib-instrument/src/instrumenter.js

```diff
@@ -80,7 +80,13 @@ class Instrumenter {
         const opts = this.opts;
         const ast = babylon.parse(code, {
             allowReturnOutsideFunction: opts.autoWrap,
-            sourceType: opts.esModules ? "module" : "script"
+            sourceType: opts.esModules ? "module" : "script",
+            plugins: [
+              'asyncGenerators',
+              'dynamicImport',
+              'flow',
+              'jsx'
+            ]
```

Comment:
@jonathanargentiero @JaKXz I don't know enough about how plugins are implemented to gauge how safe this is (@hzoo perhaps you could give this a quick 👀 ).

When are the plugins actually loaded, are they shipped as part of babylon? lol, explain this change to me like I'm five.